### PR TITLE
VIH-5383 Change participant type Solicitor to Representative

### PR DESCRIPTION
**Pull request by rehmanab**
https://tools.hmcts.net/jira/browse/VIH-5383

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts [hunk 1]
```diff
@@ -393,6 +393,7 @@ describe('AddParticipantComponent', () => {
             expect(city.value).toBe(participant.city);
             expect(county.value).toBe(participant.county);
             expect(postcode.value).toBe(participant.postcode);
+            expect(false).toBe(participant.isRepresenting);
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts:396]: shouldn't this be written as `expect(participant.isRepresenting).toBeFalsy();`

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.html [hunk 2]
```diff
@@ -33,8 +33,11 @@ <h3 class="govuk-heading-m">{{isEditMode ? 'Participants' : 'Participants added'
         <div class="govuk-grid-row vh-row">
           <div class="govuk-grid-column-two-thirds vhtable-header">
             {{pat.title}} {{pat.first_name}} {{pat.last_name}}
-            <div>
-              {{pat.hearing_role_name}}<span *ngIf="pat.representee">, representing</span>
+            <div *ngIf="!pat.isRepresenting">
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.html:36]: an ngTemplate using *ngifelse would be better suited here

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts (hunk 1 above)
Comment (rehmanab) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts]: Done

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.html (hunk 2 above)
Comment (rehmanab) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.html]: Done

**Pull request merged**